code/game/objects/structures/watercloset.dm

```diff
@@ -47,12 +51,30 @@
 	if(I.is_crowbar())
 		to_chat(user, "<span class='notice'>You start to [cistern ? "replace the lid on the cistern" : "lift the lid off the cistern"].</span>")
 		playsound(loc, 'sound/effects/stonedoor_openclose.ogg', 50, 1)
-		if(do_after(user, 30))
+		if(do_after(user, 15))
 			user.visible_message("<span class='notice'>[user] [cistern ? "replaces the lid on the cistern" : "lifts the lid off the cistern"]!</span>", "<span class='notice'>You [cistern ? "replace the lid on the cistern" : "lift the lid off the cistern"]!</span>", "You hear grinding porcelain.")
 			cistern = !cistern
 			update_icon()
 			return
 
+	if(I.is_wrench())
+		water_supply = !water_supply
+		to_chat(user, "You [water_supply ? "open" : "close"] \the [src] water supply valve. It won't refill after being flushed now.")
```

Comment:
Still says "it won't refill after being flushed now" regardless of the open/closed state.
"you open the valve, now it won't refill"
"you close the valve, now it won't refill"

Reply:
Oh i get the issue now.